packages/rich-text-editor/src/RichTextEditor/RichTextEditor.tsx

```diff
@@ -3,45 +3,56 @@ import { v4 } from "uuid"
 import classnames from "classnames"
 import { history } from "prosemirror-history"
 import { keymap } from "prosemirror-keymap"
-import { EditorState } from "prosemirror-state"
 import { Node, Schema } from "prosemirror-model"
 import { Label } from "@kaizen/draft-form"
 import { baseKeymap } from "prosemirror-commands"
-import { useRichTextEditor } from "@cultureamp/rich-text-toolkit"
+import {
+  useRichTextEditor,
+  markIsActive,
+  removeMark,
+  addMark,
+} from "@cultureamp/rich-text-toolkit"
 import { OverrideClassName } from "@kaizen/component-base"
 import { EditorContentArray } from "./types"
 import { createSchemaFromControls } from "./schema"
+import { createInitialState } from "./state"
 import { buildKeymap } from "./keymap"
 import { buildInputRules } from "./inputrules"
 import styles from "./RichTextEditor.scss"
+import { Toolbar, ToolbarSection, ToggleIconButton } from "./"
 
+type ToolbarControls = "strong" | "em" | "underline"
```

Comment:
I'm trying to remember if we decided against this for some reason, but I'm thinking we should go for `bold` and `italic` instead of `strong` and `em` to make these more user friendly.

Reply:
That sounds good to me 👍 Happy to make that change